src/main/webapp/index.html

```diff
@@ -25,7 +25,10 @@
         </text>
       </div>
       <form id="input_form">
-        <input type="text" class="input_text" id="input_text" name="input_text" onclick="empty()" value="your greating"> 
+        <div class="text-rec">
+          <input type="text" class="input_text" id="input_text" name="input_text" onclick="empty()" value="your greating">
+          <img onclick="record()" id="recording" src="/image/record.png"> 
```

Comment:
It's not a good idea to have a img element with an onclick event, it doesn't follow the ARIA practice (https://www.w3.org/TR/wai-aria-practices-1.1/) since an image is not focusable/tabbable and clickable with a keyboard.
There are other multiple ways to have a button with a custom image/icon:
```
1) <button onclick="record()"><img src="/image/record.png"></button>
2) <input type="image" src="/image/record.png" onclick="record()" />
3) <button style="background: url(/image/record.png)" ... />
```

Reply:
Okay, thank you!